docs/robots/warthog/user_manual_warthog.mdx

```diff
@@ -4,7 +4,671 @@ sidebar_position: 2
+| Solid Red                        | The MCU is not in contact with the computer. That is, the rosserial connection is not active. This condition will be seen briefly on startup while Warthog's computer is booting up. If it persists, or is seen after initialization, either the base node on the PC has crashed, the network switch has failed, or a serious MCU error has occurred. If you suspect one of these conditions, please contact our support team. |
+| Flashing Red                     | Stop circuit is broken. Twist the mushroom buttons to ensure that they're unlatched, and check any external stop hardware, if present.                                                                                                                                                                                                                                                                                         |
+| Flashing Yellow                  | Motor drivers not yet ready to drive. The motors have a brief initialization sequence which must complete after a stop condition clears before they are ready to drive. If this condition persists, please contact our support team.                                                                                                                                                                                           |
+| White Headlights, Red Taillights | When Warthog is ready to drive, the front will change from red to white. The intensity of the head and tail lights will increase slightly when actually in motion. This is the status which may be overridden by publishing your own light patterns to the `cmd_lights` ROS topic.                                                                                                                                             |
+
+### Wireless access
+
+To get Warthog connected to your local Wi-Fi, you must first access the internal computer using a wired connection.
+Open the chassis, lower the computer tray, and connect to the network port labeled STATIC with a standard ethernet cable (or connect to the Ethernet port on the exterior connector panel).
+
+#### Static IP Configuration
+
+Set your laptop's ethernet port to a static IP such as 192.168.131.101. To do this in Ubuntu, follow the steps
+below:
+
+1.  Click on the Wifi icon in the upper-right corner of your screen, and select Edit Connections
+2.  In the Network Connections window, under Ethernet, select your wired connection and then click Edit
+3.  Select the IPv4 Settings tab and then change the Method to Manual
+4.  Click the Add button to add a new address
+5.  Enter a 192.168.131.101 as the static IP under the Address column, and enter 255.255.255.0 under the NetMask column, and then select Save
+
+<center>
+  <img
+    src="/img/robot_images/warthog_images/robot_warthog_12.png"
+    width="400"
+  />
+</center>
+
+#### Connect to Warthog via SSH over Ethernet
```

Comment:
Any chance we can leverage the components/netplan_setup.mdx file for some/all of this section?

Reply:
Yes, I will update the Warthog page to use the netplan_setup Component.